meta-mender-core/recipes-mender/mender-snapshot/mender-snapshot.inc

```diff
@@ -0,0 +1,29 @@
+DESCRIPTION = "Mender tool for creating golden-images from a given installation."
+HOMEPAGE = "https://mender.io"
+
+DEPENDS:append = " glib-2.0"
+RDEPENDS:${PN} = "glib-2.0 mender-update (>= 4.0)"
+
+B = "${WORKDIR}/build"
+
+inherit go-mod
+inherit go-ptest
+
+GO_IMPORT = "github.com/mendersoftware/mender-snapshot"
```

Comment:
Once we are using `go-mod`, can't we get the rid of this `GO_IMPORT` path? Am I right that the only reason to keep this is to make it compatible with today's mender-qa script (that checks out everything under `src/github.com/mendersoftware`)?

Reply:
From what I can tell, we still can't actually. It uses it for cloning.

It was a bit surprising.